tests/ui/tests/lib/login.js

```diff
@@ -72,6 +72,11 @@ define([
                     .then(function(handles) {
 
                         return remote.switchToWindow(handles[1])
+
+                            // FOR SAFARI Added this for Safari -- it isn't finding the #authentication_email without it
+                            // Could be a page load issue, may need to add polling for the element first
+                            .sleep(5000)
```

Comment:
Could we wrap this in something similar to `this.getBrowserName(remote) === 'safari'` ?